sdks/java/extensions/sql/zetasql/src/test/java/org/apache/beam/sdk/extensions/sql/zetasql/ZetaSQLDialectSpecTest.java

```diff
@@ -144,6 +145,34 @@ public void testSimpleSelect() {
     pipeline.run().waitUntilFinish(Duration.standardMinutes(PIPELINE_EXECUTION_WAITTIME_MINUTES));
   }
 
+  @Test
+  public void testSimpleSelectWithPlannerName() {
```

Comment:
Can you add a test to show when `withQueryPlannerClass(ZetaSQLQueryPlanner.class)` is not used and BeamSqlPipelineOptions is set (to test that the ordering of finding the planner name is working).